### ISSUE TITLE
VXLAN with large MTU

### ISSUE THREAD

**Issue by RoeiGanor** [buger/goreplay#1134]
We are having a weird issue with capturing traffic with VXLAN. 
The issue is that for some requests we didnt receive response body. The setup goes as follow:
an AWS ALB forward traffic to an application vm (Apache), the ALB port is mirrored to a goreplay in the same subnet.

After some investigation we noticed that both application VM and GOR VM have large MTU (9000) and lowering the application MTU lower than ~1500 fix the issue.

We cannot keep this low mtu in our environment, so this solution is not really viable for us.

This issue might be related to #1096 .

**Comment by stephanemartin** [comment]
Yes when we have developed this feature we have fixed the MTU. I'm thinking we can add parameter to customize and manage the jumbo frames. Or I don't know yet detect the MTU.

const VxLanPacketSize = 1526 //vxlan 8 B + ethernet II 1518 B
